app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_results = dict()
```

Comment:
Why not using just -> cache_results = {}. instead of calling dict() function ?

Reply:
i thought that set would be created insted of dict